### PR TITLE
React1 week1

### PR DESCRIPTION
**Pull request by Natata08**
### The following tasks were implemented: 
1. Added a new element to the **Footer** and imported the Footer component into layout.js.
2. Added three new components: 

- **OurValues**: Displays company values in a responsive grid layout.
- **OurCrew**: Showcases team members with their photos and descriptions.
- **OurPartners**: Presents partner logos in a responsive grid layout.

### Screenshot of **/about_us** page:

screencapture-localhost-3001-about-us-2024-08-28-15_53_36

### CODE REVIEW

File: app/about_us/OurCrew.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import styles from "./OurCrew.module.css";
+
+const OurCrew = () => {
+  const crewMembers = [
```
[the remark this answers is not in the document]
Comment (ddobby94) [reply app/about_us/OurCrew.js:4]: This wasn't even part of this HW, nice job!

File: app/about_us/OurCrew.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <p className={styles["crew-description"]}>
+        Our crew is the heart and soul of Galactica. We are a diverse team of
+        seasoned space explorers, engineers, and visionaries who are united by a
+        common goal: to make space travel accessible and exciting for all.
+      </p>
+      <ul className={styles["members-list"]}>
+        {crewMembers.map((member) => {
+          return (
+            <li key={member.name} className={styles["member-item"]}>
+              <img src={member.image} alt={`photo of ${member.name}`}></img>
```
[the remark this answers is not in the document]
Comment (ddobby94) [reply app/about_us/OurCrew.js:53]: More info on the `alt` property:

- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA

File: app/about_us/OurCrew.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      position: "Crew Member",
+      image: "crew/image-anousheh-ansari.png",
+      description:
+        "Maya brings a unique blend of technical skills and customer service experience to the team. She's always ready to assist with any needs and to make sure every traveler has an unforgettable experience.",
+    },
+  ];
+
+  return (
+    <div>
+      <p className={styles["crew-description"]}>
```
[the remark this answers is not in the document]
Comment (ddobby94) [reply app/about_us/OurCrew.js:44]: So no need to change them for this PR, but for the next HW you can use this 😄

File: app/about_us/OurValues.module.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.value-description {
+  font-size: 0.8rem;
+  line-height: 1.6;
+  font-size: small;
+  font-weight: 200;
+}
+
+@media (max-width: 768px) {
```
[the remark this answers is not in the document]
Comment (ddobby94) [reply app/about_us/OurValues.module.css:37]: These numbers will repeat over and over again.
For these numbers you can register a css variable in the `globals.css` file.

```
GLOBAL .css:
:root {
  --mobile-breakpoint: 768px;
  --tablet-breakpoint: 1199px;
  ...
}

// custom CSS file for the components
@media (min-width: --mobile-breakpoint) {
  ...
}
```

**Comment by Natata08** [comment]
Thank you @ddobby94 for your review!

File: app/about_us/OurCrew.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Natata08) [reply app/about_us/OurCrew.js:53]: Thank you very much! I have never thought that it can be so important!

**Pull request merged**